parm/post_avblflds.xml

```diff
@@ -6562,22 +6562,18 @@
     <param>
        <post_avblfldidx>702</post_avblfldidx>
        <shortname>GSD_MAX_LTG_THREAT1_ON_ENTIRE_ATMOS</shortname>
```

Comment:
@EricJames-NOAA You did some modifications for ID 702 and 703. Would you consider to change their short names for consistency? such as: GSD_MAX_LTG_THREAT1_ON_SPEC_HGT_LVL_ABOVE_GRND